src/components/App/index.js

```diff
@@ -34,7 +34,7 @@ const App = () => {
                 viewport.setAttribute('content', `height=${viewheight}px, width=${viewwidth}px, initial-scale=1.0`)
             }, 300)
         }
-    }, [])
+    }, [i18n])
```

Comment:
why do we need i18n on the useEffect? maybe just window.location.pathname?

Reply:
it's an oversight. I'll fix it